tests/jeos/firstrun.pm

```diff
@@ -119,7 +119,8 @@ sub run {
     # when there are more jobs running concurrently. We need to wait for
     # various disk optimizations and snapshot enablement to land.
     # Meltdown/Spectre mitigations makes this even worse.
-    assert_screen [qw(linux-login reached-power-off)], 1000;
+    # replace linux-login by tty1-selected.
+    assert_screen [qw(tty1-selected reached-power-off)], 1000;
```

Comment:
Please use the same trick with `my $tty = get_root_console_tty;`

Reply:
no, this is not working with get_root_console_tty, it returns always tty6. and there is no root console reached yet.

Reply:
After chatting with @SergioAtSUSE, use this code, and place it in the other places where `get_root_console_tty` is being called.